subsys/bluetooth/controller/Kconfig.dtm

```diff
@@ -0,0 +1,49 @@
+# Zephyr Bluetooth Controller configuration options
+
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config BT_CTLR_DTM_HCI_SUPPORT
+	bool
+
+config BT_CTLR_DTM
+	bool
+	help
+	  Enable support for Direct Test Mode in the Controller.
+
+menuconfig BT_CTLR_DTM_HCI
+	bool "Direct Test Mode over HCI"
+	depends on BT_CTLR_DTM_HCI_SUPPORT
+	select BT_CTLR_DTM
+	help
+	  Enable support for Direct Test Mode over the HCI transport.
+
+if BT_CTLR_DTM_HCI
+
+config BT_CTLR_DTM_HCI_RX_V3
+	bool "HCI LE Receiver Test v3"
+	default y
+	help
+	  Enable support for the DTM Receiver test command v3.
+
+config BT_CTLR_DTM_HCI_TX_V3
+	bool "HCI LE Transmitter Test v3"
+	default y
+	help
+	  Enable support for the DTM Transmitter test command v3.
+
+config BT_CTLR_DTM_HCI_TX_V4
+	bool "HCI LE Transmitter Test v4"
+	default y
+	help
+	  Enable support for the DTM Transmitter test command v4.
+
+config BT_CTLR_DTM_HCI_DF_IQ_REPORT
+	bool "Connectionless IQ report HCI event"
+	depends on BT_CTLR_DF_CTE_RX
```

Comment:
Added in way suggested here. I am not sure if commands in version 3 and 4 should depends on the DF feature

Reply:
Do the tests commands require IQ reports if used in context without DF?

Reply:
The IQ reports can be generated only on the CTE reception but even then IQ reports generation is not mandatory. In case when there is no CTE in PDUs IQ reports won't be generated even when IQ reports generation is enabled in Kconfig. If your device doesn't support CTE but you won't to play with different Tx power values you can use test tx command v4 but you need to send it with CTE length set to 0 which means that CTE is disabled, if you set any other valid CTE length for such device then you get HCI not supported feature error.

The Kconfig for it looks in following way now:
```
config BT_CTLR_DTM_HCI_DF_IQ_REPORT
	bool "Connectionless IQ report HCI event [EXPERIMENTAL]"
	depends on BT_CTLR_DF_CTE_RX && BT_CTLR_DTM_HCI_RX_V3
	select EXPERIMENTAL
	help
	  Enable generation of the HCI LE Connectionless IQ Report event
	  after the Constant Tone Extension reception.
````